django/db/models/expressions.py

```diff
@@ -962,3 +986,5 @@ def asc(self):
 
     def desc(self):
         self.descending = True
+
+from django.db.models import fields
```

Comment:
Definitely -- a place specifically designed for preparing the module is needed. It'll also help 3rd party backends that need to override specific as_sql methods.